dist/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard App</title>
+  <script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+  <body>
+    <section class="leaderboard-section">
+      <h1>Leaderboard</h1>
+      <div class="leaderboard-content-container">
+        <div class="leaderboard-left-section">
+          <div class="leaderboard-left-section-title">
+            <h2>Recent scores</h2>
+            <button type="button" class="refresh-btn">Refresh</button>
+          </div>
+          <div class="score-board"></div>
+        </div>
+        <div class="leaderboard-right-section">
+          <h2>Add your score</h2>
+          <form class="leaderboard-form">
+            <input type="text" placeholder="Your name" class="input-name" />
+            <input type="text" placeholder="Your score" class="input-score"/>
+            <input type="submit" value="Submit" class="submit-btn"/>
+          </form>
+        </div>
+      </div>
+    </section>
+  </body>
+</html>
```

Comment:
- [ ] It would be better to connect the JavaScript code to the HTML so that the expected functionality can be met. Currently, I'm unable send data to the API, unable to see that data from the API displayed on the browser. Kindly fix this before asking for another code review request 
![Screenshot from 2023-06-03 00-23-59](https://github.com/Simpleshaikh1/LeaderBoard/assets/46388113/907651ab-b2e6-48b7-a00b-914b27ccfbc8)
![Screenshot from 2023-06-03 00-23-42](https://github.com/Simpleshaikh1/LeaderBoard/assets/46388113/53f6396a-0dc7-4dc5-9595-2b39834f4399)
Try to add some space to the `Add your score` text and the inputs